nav/nav_stack/config/caffeine_costmap_common_params.yaml

```diff
@@ -0,0 +1,9 @@
+obstacle_range: 5.5
+raytrace_range: 6.0
+
+footprint: [[-0.734, -0.582], [-0.734, 0.582], [0.734, 0.582], [0.734, -0.582]]
+inflation_radius: 1.0
+
+observation_sources: laser_scan_sensor
+
+laser_scan_sensor: {sensor_frame: caffeine/base_laser, data_type: LaserScan, topic: scan, marking: true, clearing: true}
```

Comment:
We are missing a `point_cloud_sensor` observation source (I think that is what it is called). In the wiki I added concerning [Understanding the Navigation Stack](https://github.com/UTRA-ART/Caffeine/wiki/Understanding-the-Navigation-Stack), this reference can be found for observation sources: http://wiki.ros.org/costmap_2d/hydro/obstacles